### PR TITLE
Feat/integrate wandb

### PR DESCRIPTION
**Pull request by V0XNIHILI**
Very much work in progress, but makes it easier to see the branch diff

### CODE REVIEW

File: src/model/mybag_model.py [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+        return self.mlb.fit_transform(y)
+
+    def train(self, X_train, y_train):
+        self.classifier.fit(X_train, y_train)
+
+    def predict(self, X_test):
+        featurized = self.get_features(X_test)
+        return self.classifier.predict(featurized)
+
+    def save(self, path):
```
Comment (V0XNIHILI) [inline src/model/mybag_model.py]: Maybe this can be part of the Base class? This call I mean to dump it

Comment (adondera) [reply src/model/mybag_model.py]: Done, good suggestion

**Comment by V0XNIHILI** [comment]
I will continue tonight to finish this PR, hopefully we can merge it then before the meeting with Luis

**Comment by V0XNIHILI** [comment]
@V0XNIHILI: Still need to fix pytest not working

**Review by adondera — APPROVED** [review]

**Review by farsene — APPROVED** [review]

**Pull request merged**